docs/01_manual.md

```diff
@@ -26,7 +26,7 @@ SYNOPSIS
                      | get [-s,--skip-completion] [-d,--directory <dir>] <item-id> 
                      | stream [-f,--format zip|tar] <item-id>
                      | enum [[-i,--inactive|-a,--all] <bag-id>]
-                     | locate [<item-id>]
+                     | locate [-f,--file-data-location] <item-id>
```

Comment:
If `-f` is specified the "real location" of the file is looked up, if the `item-id` points to a file.